lti_emailer/canvas_api_client.py

```diff
@@ -21,6 +21,21 @@
 SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
 
 
+def get_section(canvas_course_id, section_id):
```

Comment:
I'm really not a fan of the "grab piles of stuff we might not need, cache it, and hope we use the rest of it later" approach.